app/views/layouts/application.html.erb

```diff
@@ -8,12 +8,41 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
-  <div class="containter">
+  <div class="container">
     <ul class="nav nav-tabs">
       <li><%= link_to "Home", root_path %></li>
       <li><%= link_to "About", about_path %></li>
     </ul>
 
+    <div class="container">
+      <% if user_signed_in? %>
+        Signed in as: <strong><%= current_user.email %></strong> |
+        <%= link_to "Edit Profile", edit_user_registration_path, class: 'navbar-link' %> - <%= link_to "Sign Out", destroy_user_session_path, method: :delete, class: 'navbar-link' %>
+      <% else %>
+        <%= link_to "Sign Up", new_user_registration_path, class: 'navbar-link' %> -
+        <%= link_to "Sign In", new_user_session_path, class: 'navbar-link' %>
+      <% end %>
+    </div>
+
+    <div class="flash">
```

Comment:
Its unlikely but possible for multiple level flash messages to be set for one request (e.g. flash notice and alert levels both have messages).

If a flash[:notice] and a flash[:error] were both set during the same request, what would lines 28-43 show?